### DISCUSSION TITLE
FormGrid布局问题

### DISCUSSION THREAD

**Discussion by caixianglin** [alibaba/formily#2754] [Q&A 2.x] [answered]
https://codesandbox.io/s/strange-kalam-6144y?file=/App.tsx
![image](https://user-images.githubusercontent.com/10279815/149057558-f977e824-cb53-463d-be66-b9ee58930c7b.png)

**Comment by janryWang** [comment, marked as answer]
配置
```
"x-component-props": {
        maxColumns: 3,
        minColumns: 2,
        strictAutoFit:true,
      },
```
